x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -410,7 +410,7 @@ const UnstyledProcessEventDot = React.memo(
               alignSelf: 'flex-start',
               background: colorMap.resolverBackground,
               display: `${isShowingEventActions ? 'flex' : 'none'}`,
-              margin: 0,
+              margin: '2px 0 0 0',
```

Comment:
:grey_question: This kicks it down a hairline, right? I got it to stop doing the rendering thing (But I didn't test it very much) by setting `contain: layout` on the button above. Maybe something to keep in mind later if someone wants to nudge it back where it was. It's a weird issue and AFAICT one of those things that "shouldn't happen" by the book.

Reply:
Yea it does. The padding at the top of the popover was `2px` so I added the margin to account for that. Yea, it shouldn't because even with the padding, it should still respect each elements space in the layout. 🤷‍♂️